oci/handler/directory.go

```diff
@@ -410,21 +414,38 @@ func (handle *DirectoryHandler) FetchImage(ctx context.Context, fullref, platfor
 	return nil
 }
 
+// directoryHandlerAuthorization is used handle looking up the already populated
```

Comment:
`handle` -> `to handle`

I can't stop correcting typos

Reply:
I need to install a spell checker, thanks!